lt-server/src/models/donation.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose');
+
+const donationSchema = new mongoose.Schema(
+    {
+        donor: {
+            type: mongoose.Types.ObjectId,
+            ref: 'User',
+        },
+        amount: {
+            type: Number,
+            default: 0,
+            required: true,
+        },
+        dateOfDonation: {
```

Comment:
since you added `timestamp: true`, we don't need `dateOfDonation`

Reply:
timestamp removed